trunk/SUAVE/Plots/Performance/Mission_Plots.py

```diff
@@ -1640,370 +1772,370 @@ def create_video_frames(results,vehicle, save_figure = True ,flight_profile = Tr
             total_microphone_locations          - microphone locations
-            
-    Outputs: 
+
+    Outputs:
     Plots
-    
+
     Properties Used:
-    N/A	
-    """        
-    # unpack 
+    N/A
+    """
+    # unpack
     dim_segs     = len(results.segments)
     dim_gm       = results.segments[0].conditions.noise.number_ground_microphones
-    dim_ctrl_pts = len(results.segments[0].conditions.frames.inertial.time[:,0])  
-    N_gm_x       = results.segments[0].analyses.noise.settings.level_ground_microphone_x_resolution  
-    N_gm_y       = results.segments[0].analyses.noise.settings.level_ground_microphone_y_resolution 
+    dim_ctrl_pts = len(results.segments[0].conditions.frames.inertial.time[:,0])
+    N_gm_x       = results.segments[0].analyses.noise.settings.level_ground_microphone_x_resolution
+    N_gm_y       = results.segments[0].analyses.noise.settings.level_ground_microphone_y_resolution
     gm           = results.segments[0].conditions.noise.ground_microphone_locations[0].reshape(N_gm_x,N_gm_y,3)
     gm_x         = -gm[:,:,0]
     gm_y         = -gm[:,:,1]
-    colors       = cm.jet(np.linspace(0, 1,N_gm_y))   
-    
+    colors       = cm.jet(np.linspace(0, 1,N_gm_y))
+
     # figure parameters
-    axis_font    = {'size':'14'} 
+    axis_font    = {'size':'14'}
     fig          = plt.figure(save_filename)
-    fig.set_size_inches(10, 8) 
-    axes        = fig.add_subplot(1,1,1) 
-     
+    fig.set_size_inches(10, 8)
+    axes        = fig.add_subplot(1,1,1)
+
     SPL = np.zeros((dim_segs,dim_ctrl_pts,N_gm_x,N_gm_y))
-    # loop through control points 
-    for i in range(dim_segs):  
+    # loop through control points
+    for i in range(dim_segs):
         for j in range(dim_ctrl_pts):
             if results.segments[i].battery_discharge == False:
-                pass 
+                pass
             else:
-                SPL[i,j,:] = results.segments[i].conditions.noise.total_SPL_dBA[j,:dim_gm].reshape(N_gm_x,N_gm_y)  
-    max_SPL = np.max(np.max(SPL,axis=0),axis=0)   
-    for k in range(N_gm_y):    
-        axes.plot(gm_x[:,0]/Units.nmi, max_SPL[:,k], marker = 'o', color = colors[k], label= r'mic at y = ' + str(round(gm_y[0,k],1)) + r' m' ) 
+                SPL[i,j,:] = results.segments[i].conditions.noise.total_SPL_dBA[j,:dim_gm].reshape(N_gm_x,N_gm_y)
+    max_SPL = np.max(np.max(SPL,axis=0),axis=0)
+    for k in range(N_gm_y):
+        axes.plot(gm_x[:,0]/Units.nmi, max_SPL[:,k], marker = 'o', color = colors[k], label= r'mic at y = ' + str(round(gm_y[0,k],1)) + r' m' )
     axes.set_ylabel('SPL (dBA)',axis_font)
-    axes.set_xlabel('Range (nmi)',axis_font)  
+    axes.set_xlabel('Range (nmi)',axis_font)
     set_axes(axes)
-    axes.legend(loc='upper right')         
+    axes.legend(loc='upper right')
     if save_figure:
-        plt.savefig(save_filename + ".png")  
-        
+        plt.savefig(save_filename + ".png")
+
 
     return
 
 
 ## @defgroup Plots-Performance
```

Comment:
this one still says defgroup

Reply:
You can ignore this comment and make an issue if you want instead

Reply:
fixed